webapp/pages/profile/_id/_slug.vue

```diff
@@ -430,6 +430,14 @@ export default {
         },
         fetchPolicy: 'cache-and-network'
       })
+    },
+    userName(maxLength) {
+      // Return Anonymous if no Username is given
+      if (!this.user.name) {
```

Comment:
```suggestion
  return this.user.name || this.$t('profile.userAnonym')
```

Reply:
codewise not recomended since we have a possible truncate on user.name. Therefore it is wise to check if user.name is present unless we want to have `|| this.$t('profile.userAnonym')` duplicated and a very complicated statement.

```
export default {
  methods: {
    userName(userName, maxLength) {
      // Return Anonymous if no Username is given
      if (!userName) {
        return this.$t('profile.userAnonym')
      }
      // Return full Username or truncated Username
      return maxLength ? this.$filters.truncate(userName, maxLength) : userName
    }
  }
}
```

Suggestion: no triage

Reply:
? :thinking: if you truncate the `userName` the value cannot be falsy.

My suggestion: In case you are not so sure if a situation can ever happen, write a test for it and see! :+1:

Reply:
Ahh now I get it. Yes, the best is a guard clause, like you wrote it. Just put it on one line:
```js
if (!userName) return this.$t('profile.userAnonym')
```

Reply:
That's not lint compatible (most likely)